packages/n/neovide/files/neovide.desktop

```diff
@@ -0,0 +1,8 @@
+[Desktop Entry]
```

Comment:
This file should be included in the sources: https://github.com/neovide/neovide/blob/main/assets/neovide.desktop

Reply:
Yes, `Icon=neovide.png` won't work with icon themes. The .desktop file shipped with the source looks fine, just use it:
```
install -Dm00644 assets/neovide.desktop $installdir/usr/share/applications/dev.neovide.Neovide.desktop
```
Note that we're changing the name of the desktop file to the fully qualified one, so this needs a corresponding patch so that window detection works. I went ahead and created this patch for you [here](https://pastebin.com/raw/6BsD2LLL), download it and copy it to the files subfolder and then add `%patch -p1 -i $pkgfiles/0001-Use-fully-qualified-app_id.patch` to your setup phase.